firebase/firestore.indexes.json

```diff
@@ -1,4 +1,12 @@
 {
-  "indexes": [],
+  "indexes": [
+    {
+      "collectionId": "requests",
+      "fields": [
+        { "fieldPath": "status", "mode": "ASCENDING" },
+        { "fieldPath": "latLng", "mode": "ASCENDING" }
+      ]
+    }
+  ],
   "fieldOverrides": []
```

Comment:
Can we also add the code to create the other index we needed (there were two indexes used).

Reply:
that index is no longer being used. that was created initially